2023-07-11 13:58:14,708 [+] pydumpck is a multi-threads tool for decompile exe,elf,pyz,pyc packed by python which is base on pycdc and uncompyle6.sometimes its py-file result not exactly right ,maybe could use uncompyle6.
--------------------
pydumpck initilizing with 1.16.0 :pyinstaller_dump.py:115-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyinstaller_dump.py on(pyinstaller_dump.run) at 100@MainThread  
2023-07-11 13:58:14,711 [*] plugins loaded with ['uncompyle6'] :__init__.py:147-10  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_common_dump/__init__.py on(__init__.load_plugins) at 100@MainThread  
2023-07-11 13:58:14,714 [-] attention! when use uncompyle6 , you should use --decompile_file specified which file to decompile for faster task. :__init__.py:151-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_common_dump/__init__.py on(__init__.load_plugins) at 100@MainThread  
2023-07-11 13:58:20,855 [+] pydumpck is a multi-threads tool for decompile exe,elf,pyz,pyc packed by python which is base on pycdc and uncompyle6.sometimes its py-file result not exactly right ,maybe could use uncompyle6.
--------------------
pydumpck initilizing with 1.16.0 :pyinstaller_dump.py:115-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyinstaller_dump.py on(pyinstaller_dump.run) at 131@MainThread  
2023-07-11 13:58:20,857 [*] plugins loaded with ['uncompyle6'] :__init__.py:147-10  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_common_dump/__init__.py on(__init__.load_plugins) at 131@MainThread  
2023-07-11 13:58:20,859 [-] attention! when use uncompyle6 , you should use --decompile_file specified which file to decompile for faster task. :__init__.py:151-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_common_dump/__init__.py on(__init__.load_plugins) at 131@MainThread  
2023-07-11 13:58:23,865 [*] target file input:pyreverser.exe
to:output_1043481 :__init__.py:182-10  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_common_dump/__init__.py on(__init__.main) at 131@MainThread  
2023-07-11 13:58:24,855 [*] start dump target file. type:arch :__init__.py:195-10  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_common_dump/__init__.py on(__init__.main) at 131@MainThread  
2023-07-11 13:58:27,888 [+] struct file found:b'' :__init__.py:136-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.add) at 131@MainThread  
2023-07-11 13:58:27,891 [*] 
export pyc :__init__.py:94-10  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.progress_check) at 131@MainThread  
2023-07-11 13:58:31,157 [*] 
decompile pyc :__init__.py:96-10  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.progress_check) at 131@MainThread  
2023-07-11 13:58:31,181 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_0  
2023-07-11 13:58:31,181 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_5  
2023-07-11 13:58:31,182 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_4  
2023-07-11 13:58:31,183 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_3  
2023-07-11 13:58:31,184 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_1  
2023-07-11 13:58:31,184 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_6  
2023-07-11 13:58:31,185 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_7  
2023-07-11 13:58:31,186 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_2  
2023-07-11 13:58:31,308 [-] decompile bytecode by uncompyle6 fail file:output_1043481/pyiboot01_bootstrap.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_5  
2023-07-11 13:58:31,308 [-] decompile bytecode by uncompyle6 fail file:output_1043481/struct.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_0  
2023-07-11 13:58:31,308 [-] decompile bytecode by uncompyle6 fail file:output_1043481/pyimod04_pywin32.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_4  
2023-07-11 13:58:31,308 [-] decompile bytecode by uncompyle6 fail file:output_1043481/pyimod03_ctypes.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_3  
2023-07-11 13:58:31,312 [-] decompile bytecode by uncompyle6 fail file:output_1043481/pyi_rth_inspect.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_6  
2023-07-11 13:58:31,312 [-] decompile bytecode by uncompyle6 fail file:output_1043481/pyreverser.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_7  
2023-07-11 13:58:31,314 [!] fail to decompile bytecode on file:output_1043481/pyiboot01_bootstrap.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_5  
2023-07-11 13:58:31,318 [-] decompile bytecode by uncompyle6 fail file:output_1043481/pyimod01_archive.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_1  
2023-07-11 13:58:31,322 [!] fail to decompile bytecode on file:output_1043481/struct.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_0  
2023-07-11 13:58:31,322 [-] decompile bytecode by uncompyle6 fail file:output_1043481/pyimod02_importers.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_2  
2023-07-11 13:58:31,326 [!] fail to decompile bytecode on file:output_1043481/pyimod04_pywin32.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_4  
2023-07-11 13:58:31,329 [!] fail to decompile bytecode on file:output_1043481/pyimod03_ctypes.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_3  
2023-07-11 13:58:31,334 [!] fail to decompile bytecode on file:output_1043481/pyi_rth_inspect.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_6  
2023-07-11 13:58:31,337 [!] fail to decompile bytecode on file:output_1043481/pyreverser.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_7  
2023-07-11 13:58:31,342 [!] fail to decompile bytecode on file:output_1043481/pyimod01_archive.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_1  
2023-07-11 13:58:31,347 [!] fail to decompile bytecode on file:output_1043481/pyimod02_importers.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_2  
2023-07-11 13:58:32,163 [*] 
extract pyz :__init__.py:98-10  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.progress_check) at 131@MainThread  
2023-07-11 13:58:32,177 [+] Found 97 files in PYZ archive :pyz_extract.py:60-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyz_extract.py on(pyz_extract._handle_file) at 131@MainThread  
2023-07-11 13:58:34,496 [*] 
decompile pyc for `extract pyz` :__init__.py:100-10  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.progress_check) at 131@MainThread  
2023-07-11 13:58:34,620 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_5  
2023-07-11 13:58:34,623 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_11  
2023-07-11 13:58:34,628 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_2  
2023-07-11 13:58:34,630 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_1  
2023-07-11 13:58:34,636 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_12  
2023-07-11 13:58:34,637 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_18  
2023-07-11 13:58:34,646 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_7  
2023-07-11 13:58:34,667 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_33  
2023-07-11 13:58:34,675 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_4  
2023-07-11 13:58:34,677 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_10  
2023-07-11 13:58:34,682 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_6  
2023-07-11 13:58:34,688 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_22  
2023-07-11 13:58:34,691 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_23  
2023-07-11 13:58:34,693 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_14  
2023-07-11 13:58:34,698 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_16  
2023-07-11 13:58:34,700 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_29  
2023-07-11 13:58:34,706 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_8  
2023-07-11 13:58:34,707 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_20  
2023-07-11 13:58:34,709 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_15  
2023-07-11 13:58:34,714 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_31  
2023-07-11 13:58:34,714 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_13  
2023-07-11 13:58:34,715 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_21  
2023-07-11 13:58:34,717 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_28  
2023-07-11 13:58:34,718 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_24  
2023-07-11 13:58:34,725 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_17  
2023-07-11 13:58:34,728 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_35  
2023-07-11 13:58:34,731 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_0  
2023-07-11 13:58:34,733 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_30  
2023-07-11 13:58:34,735 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_40  
2023-07-11 13:58:34,736 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_3  
2023-07-11 13:58:34,739 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_39  
2023-07-11 13:58:34,741 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_36  
2023-07-11 13:58:34,743 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_27  
2023-07-11 13:58:34,745 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_9  
2023-07-11 13:58:34,746 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_19  
2023-07-11 13:58:34,748 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_34  
2023-07-11 13:58:34,750 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_32  
2023-07-11 13:58:34,752 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_26  
2023-07-11 13:58:34,753 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_25  
2023-07-11 13:58:34,755 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_42  
2023-07-11 13:58:34,757 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_37  
2023-07-11 13:58:34,759 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_38  
2023-07-11 13:58:34,767 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_41  
2023-07-11 13:58:34,772 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_43  
2023-07-11 13:58:34,786 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_45  
2023-07-11 13:58:34,789 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_46  
2023-07-11 13:58:34,792 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_44  
2023-07-11 13:58:34,810 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_47  
2023-07-11 13:58:34,846 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_48  
2023-07-11 13:58:34,850 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_49  
2023-07-11 13:58:34,873 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_55  
2023-07-11 13:58:34,881 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_50  
2023-07-11 13:58:35,535 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/_compression.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_8  
2023-07-11 13:58:34,887 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_51  
2023-07-11 13:58:34,892 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_64  
2023-07-11 13:58:34,895 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_52  
2023-07-11 13:58:34,900 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_58  
2023-07-11 13:58:34,903 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_57  
2023-07-11 13:58:34,915 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_61  
2023-07-11 13:58:34,919 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_53  
2023-07-11 13:58:34,920 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_70  
2023-07-11 13:58:34,923 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_62  
2023-07-11 13:58:34,932 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_60  
2023-07-11 13:58:34,943 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_67  
2023-07-11 13:58:34,946 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_63  
2023-07-11 13:58:34,953 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_59  
2023-07-11 13:58:34,962 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_71  
2023-07-11 13:58:34,959 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_66  
2023-07-11 13:58:34,971 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_68  
2023-07-11 13:58:34,976 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_56  
2023-07-11 13:58:34,988 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_72  
2023-07-11 13:58:34,990 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_69  
2023-07-11 13:58:34,993 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_65  
2023-07-11 13:58:35,004 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/bz2.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_11  
2023-07-11 13:58:36,059 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/bz2.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_11  
2023-07-11 13:58:35,015 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_75  
2023-07-11 13:58:35,023 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/_compat_pickle.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_5  
2023-07-11 13:58:35,026 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_74  
2023-07-11 13:58:35,038 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_76  
2023-07-11 13:58:35,043 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_77  
2023-07-11 13:58:35,056 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_79  
2023-07-11 13:58:35,060 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_78  
2023-07-11 13:58:35,066 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_80  
2023-07-11 13:58:35,072 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_81  
2023-07-11 13:58:35,074 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/calendar.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_2  
2023-07-11 13:58:36,309 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/calendar.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_2  
2023-07-11 13:58:35,091 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_82  
2023-07-11 13:58:35,097 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_83  
2023-07-11 13:58:35,108 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_87  
2023-07-11 13:58:35,109 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_84  
2023-07-11 13:58:35,112 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_86  
2023-07-11 13:58:35,123 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_85  
2023-07-11 13:58:35,130 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_88  
2023-07-11 13:58:35,133 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/decimal.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_18  
2023-07-11 13:58:35,141 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/contextlib.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_12  
2023-07-11 13:58:35,150 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/ast.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_7  
2023-07-11 13:58:35,161 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_89  
2023-07-11 13:58:35,163 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_91  
2023-07-11 13:58:35,180 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_90  
2023-07-11 13:58:35,197 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_93  
2023-07-11 13:58:35,198 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_92  
2023-07-11 13:58:35,199 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_95  
2023-07-11 13:58:35,206 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/email/headerregistry.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_33  
2023-07-11 13:58:35,215 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_94  
2023-07-11 13:58:35,239 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/_pydecimal.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_4  
2023-07-11 13:58:35,241 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/base64.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_10  
2023-07-11 13:58:35,248 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/email/_header_value_parser.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_22  
2023-07-11 13:58:35,256 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/argparse.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_6  
2023-07-11 13:58:35,296 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/email/_parseaddr.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_23  
2023-07-11 13:58:35,314 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/copy.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_14  
2023-07-11 13:58:35,372 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/dataclasses.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_16  
2023-07-11 13:58:35,530 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/csv.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_15  
2023-07-11 13:58:35,535 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/contextvars.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_13  
2023-07-11 13:58:34,883 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_54  
2023-07-11 13:58:35,558 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/_compression.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_8  
2023-07-11 13:58:35,582 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/email/errors.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_29  
2023-07-11 13:58:35,818 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/email/__init__.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_20  
2023-07-11 13:58:35,836 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/email/generator.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_31  
2023-07-11 13:58:35,883 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/datetime.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_17  
2023-07-11 13:58:35,915 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/_py_abc.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_0  
2023-07-11 13:58:35,934 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/email/encoders.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_28  
2023-07-11 13:58:35,974 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/email/_encoded_words.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_21  
2023-07-11 13:58:35,991 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/fractions.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_40  
2023-07-11 13:58:36,005 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/_threading_local.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_3  
2023-07-11 13:58:36,005 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/email/header.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_32  
2023-07-11 13:58:36,005 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/email/message.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_35  
2023-07-11 13:58:36,014 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/email/feedparser.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_30  
2023-07-11 13:58:36,039 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/email/iterators.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_34  
2023-07-11 13:58:36,042 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/email/contentmanager.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_27  
2023-07-11 13:58:36,048 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/email/_policybase.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_24  
2023-07-11 13:58:35,011 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_73  
2023-07-11 13:58:36,080 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/http/__init__.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_47  
2023-07-11 13:58:36,086 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/email/charset.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_26  
2023-07-11 13:58:36,087 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/email/parser.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_36  
2023-07-11 13:58:36,089 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/email/policy.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_37  
2023-07-11 13:58:36,089 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/email/base64mime.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_25  
2023-07-11 13:58:36,093 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/email/utils.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_39  
2023-07-11 13:58:36,093 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/dis.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_19  
2023-07-11 13:58:36,095 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/getopt.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_42  
2023-07-11 13:58:36,107 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/http/cookiejar.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_49  
2023-07-11 13:58:36,109 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/email/quoprimime.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_38  
2023-07-11 13:58:36,117 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/http/client.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_48  
2023-07-11 13:58:36,117 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/_strptime.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_9  
2023-07-11 13:58:36,151 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/gzip.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_45  
2023-07-11 13:58:36,155 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/getpass.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_43  
2023-07-11 13:58:36,159 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/ftplib.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_41  
2023-07-11 13:58:36,160 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/_compat_pickle.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_5  
2023-07-11 13:58:36,219 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/importlib/util.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_64  
2023-07-11 13:58:36,221 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/importlib/__init__.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_50  
2023-07-11 13:58:36,224 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/importlib/_bootstrap.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_52  
2023-07-11 13:58:36,224 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/importlib/machinery.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_55  
2023-07-11 13:58:36,244 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/importlib/_abc.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_51  
2023-07-11 13:58:36,250 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/importlib/readers.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_63  
2023-07-11 13:58:36,257 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/importlib/_bootstrap_external.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_53  
2023-07-11 13:58:36,272 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/hashlib.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_46  
2023-07-11 13:58:36,272 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/gettext.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_44  
2023-07-11 13:58:36,284 [+] find suitable python-version:(3, 10),magic:3439 :__init__.py:41-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/pyc/__init__.py on(__init__.check_marshal) at 131@ThreadPoolExecutor-0_11  
2023-07-11 13:58:36,286 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/importlib/metadata/_adapters.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_57  
2023-07-11 13:58:36,297 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/importlib/metadata/_collections.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_58  
2023-07-11 13:58:35,075 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/bisect.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_1  
2023-07-11 13:58:36,314 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/importlib/metadata/_text.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_62  
2023-07-11 13:58:36,320 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/importlib/metadata/_functools.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_59  
2023-07-11 13:58:36,320 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/importlib/metadata/_itertools.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_60  
2023-07-11 13:58:36,322 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/importlib/metadata/_meta.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_61  
2023-07-11 13:58:36,342 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/importlib/metadata/__init__.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_56  
2023-07-11 13:58:36,348 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/logging/__init__.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_66  
2023-07-11 13:58:36,385 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/decimal.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_18  
2023-07-11 13:58:36,390 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/contextlib.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_12  
2023-07-11 13:58:36,395 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/ast.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_7  
2023-07-11 13:58:36,423 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/nturl2path.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_70  
2023-07-11 13:58:36,445 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/email/headerregistry.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_33  
2023-07-11 13:58:36,456 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/_pydecimal.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_4  
2023-07-11 13:58:36,461 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/base64.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_10  
2023-07-11 13:58:36,466 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/email/_header_value_parser.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_22  
2023-07-11 13:58:36,470 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/argparse.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_6  
2023-07-11 13:58:36,475 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/email/_parseaddr.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_23  
2023-07-11 13:58:36,484 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/copy.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_14  
2023-07-11 13:58:36,490 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/dataclasses.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_16  
2023-07-11 13:58:36,495 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/csv.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_15  
2023-07-11 13:58:36,509 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/contextvars.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_13  
2023-07-11 13:58:36,528 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/lzma.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_67  
2023-07-11 13:58:36,530 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/numbers.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_71  
2023-07-11 13:58:36,532 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/email/errors.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_29  
2023-07-11 13:58:36,537 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/email/__init__.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_20  
2023-07-11 13:58:36,542 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/email/generator.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_31  
2023-07-11 13:58:36,546 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/datetime.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_17  
2023-07-11 13:58:36,550 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/_py_abc.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_0  
2023-07-11 13:58:36,554 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/email/encoders.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_28  
2023-07-11 13:58:36,558 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/email/_encoded_words.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_21  
2023-07-11 13:58:36,562 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/fractions.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_40  
2023-07-11 13:58:36,567 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/_threading_local.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_3  
2023-07-11 13:58:36,570 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/email/header.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_32  
2023-07-11 13:58:36,574 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/email/message.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_35  
2023-07-11 13:58:36,578 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/email/feedparser.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_30  
2023-07-11 13:58:36,582 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/email/iterators.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_34  
2023-07-11 13:58:36,585 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/email/contentmanager.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_27  
2023-07-11 13:58:36,589 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/email/_policybase.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_24  
2023-07-11 13:58:36,594 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/http/__init__.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_47  
2023-07-11 13:58:36,608 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/email/charset.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_26  
2023-07-11 13:58:36,618 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/email/parser.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_36  
2023-07-11 13:58:36,623 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/email/policy.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_37  
2023-07-11 13:58:36,627 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/netrc.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_69  
2023-07-11 13:58:36,631 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/opcode.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_72  
2023-07-11 13:58:36,632 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/mimetypes.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_68  
2023-07-11 13:58:36,632 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/email/base64mime.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_25  
2023-07-11 13:58:36,635 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/email/utils.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_39  
2023-07-11 13:58:36,638 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/dis.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_19  
2023-07-11 13:58:36,644 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/getopt.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_42  
2023-07-11 13:58:36,647 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/http/cookiejar.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_49  
2023-07-11 13:58:36,650 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/email/quoprimime.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_38  
2023-07-11 13:58:36,662 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/http/client.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_48  
2023-07-11 13:58:36,686 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/_strptime.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_9  
2023-07-11 13:58:36,706 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/gzip.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_45  
2023-07-11 13:58:36,731 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/getpass.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_43  
2023-07-11 13:58:36,732 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/pickle.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_74  
2023-07-11 13:58:36,732 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/inspect.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_65  
2023-07-11 13:58:36,741 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/pprint.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_75  
2023-07-11 13:58:36,743 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/ftplib.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_41  
2023-07-11 13:58:36,745 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/random.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_78  
2023-07-11 13:58:36,756 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/shlex.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_80  
2023-07-11 13:58:36,762 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/quopri.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_77  
2023-07-11 13:58:36,764 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/py_compile.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_76  
2023-07-11 13:58:36,778 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/selectors.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_79  
2023-07-11 13:58:36,784 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/importlib/util.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_64  
2023-07-11 13:58:36,792 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/uu.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_95  
2023-07-11 13:58:36,817 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/typing.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_94  
2023-07-11 13:58:36,819 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/signal.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_82  
2023-07-11 13:58:36,820 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/socket.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_83  
2023-07-11 13:58:36,820 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/shutil.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_81  
2023-07-11 13:58:36,822 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/ssl.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_84  
2023-07-11 13:58:36,826 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/string.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_86  
2023-07-11 13:58:36,832 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/importlib/__init__.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_50  
2023-07-11 13:58:36,834 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/statistics.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_85  
2023-07-11 13:58:36,837 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/tracemalloc.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_93  
2023-07-11 13:58:36,842 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/subprocess.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_88  
2023-07-11 13:58:36,848 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/tarfile.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_89  
2023-07-11 13:58:36,849 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/stringprep.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_87  
2023-07-11 13:58:36,855 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/tempfile.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_90  
2023-07-11 13:58:36,856 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/textwrap.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_91  
2023-07-11 13:58:36,857 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/importlib/_bootstrap.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_52  
2023-07-11 13:58:36,860 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/threading.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_92  
2023-07-11 13:58:36,861 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/importlib/machinery.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_55  
2023-07-11 13:58:36,866 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/optparse.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_73  
2023-07-11 13:58:36,867 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/importlib/_abc.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_51  
2023-07-11 13:58:36,871 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/importlib/readers.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_63  
2023-07-11 13:58:36,875 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/importlib/_bootstrap_external.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_53  
2023-07-11 13:58:36,879 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/hashlib.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_46  
2023-07-11 13:58:36,881 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/importlib/abc.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_54  
2023-07-11 13:58:36,884 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/gettext.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_44  
2023-07-11 13:58:36,893 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/importlib/metadata/_adapters.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_57  
2023-07-11 13:58:36,897 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/importlib/metadata/_collections.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_58  
2023-07-11 13:58:36,900 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/bisect.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_1  
2023-07-11 13:58:36,904 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/importlib/metadata/_text.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_62  
2023-07-11 13:58:36,908 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/importlib/metadata/_functools.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_59  
2023-07-11 13:58:36,912 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/importlib/metadata/_itertools.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_60  
2023-07-11 13:58:36,918 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/importlib/metadata/_meta.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_61  
2023-07-11 13:58:36,922 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/importlib/metadata/__init__.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_56  
2023-07-11 13:58:36,926 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/logging/__init__.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_66  
2023-07-11 13:58:36,941 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/nturl2path.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_70  
2023-07-11 13:58:36,951 [-] decompile bytecode by uncompyle6 fail file:output_1043481/PYZ-00.pyz_extract/zipfile.pyc ,with error :Unsupported Python version, 3.10.0, for decompilation :__init__.py:90-40  /home/halston/.local/lib/python3.10/site-packages/pydumpck/pyc_checker/__init__.py on(__init__.exec_uncompyle6) at 131@ThreadPoolExecutor-0_11  
2023-07-11 13:58:36,982 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/lzma.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_67  
2023-07-11 13:58:36,985 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/numbers.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_71  
2023-07-11 13:58:37,038 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/netrc.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_69  
2023-07-11 13:58:37,048 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/opcode.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_72  
2023-07-11 13:58:37,052 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/mimetypes.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_68  
2023-07-11 13:58:37,079 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/pickle.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_74  
2023-07-11 13:58:37,081 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/inspect.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_65  
2023-07-11 13:58:37,083 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/pprint.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_75  
2023-07-11 13:58:37,090 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/random.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_78  
2023-07-11 13:58:37,094 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/shlex.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_80  
2023-07-11 13:58:37,096 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/quopri.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_77  
2023-07-11 13:58:37,099 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/py_compile.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_76  
2023-07-11 13:58:37,101 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/selectors.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_79  
2023-07-11 13:58:37,107 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/uu.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_95  
2023-07-11 13:58:37,111 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/typing.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_94  
2023-07-11 13:58:37,114 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/signal.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_82  
2023-07-11 13:58:37,116 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/socket.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_83  
2023-07-11 13:58:37,118 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/shutil.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_81  
2023-07-11 13:58:37,121 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/ssl.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_84  
2023-07-11 13:58:37,124 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/string.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_86  
2023-07-11 13:58:37,130 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/statistics.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_85  
2023-07-11 13:58:37,133 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/tracemalloc.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_93  
2023-07-11 13:58:37,135 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/subprocess.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_88  
2023-07-11 13:58:37,138 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/tarfile.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_89  
2023-07-11 13:58:37,141 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/stringprep.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_87  
2023-07-11 13:58:37,144 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/tempfile.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_90  
2023-07-11 13:58:37,146 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/textwrap.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_91  
2023-07-11 13:58:37,152 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/threading.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_92  
2023-07-11 13:58:37,158 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/optparse.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_73  
2023-07-11 13:58:37,172 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/importlib/abc.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_54  
2023-07-11 13:58:37,204 [!] fail to decompile bytecode on file:output_1043481/PYZ-00.pyz_extract/zipfile.pyc,with error:pycdc not initilized :__init__.py:41-30  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.decompile_pyc) at 131@ThreadPoolExecutor-0_11  
2023-07-11 13:58:38,144 [*] 
progress_check completed :__init__.py:102-10  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_package/package_struct/__init__.py on(__init__.progress_check) at 131@MainThread  
2023-07-11 13:58:38,146 [+] completed,cost 17290ms with result:16 arch file(s) handled. :__init__.py:159-20  /home/halston/.local/lib/python3.10/site-packages/pydumpck/py_common_dump/__init__.py on(__init__.statistics_status) at 131@MainThread  
